code/modules/clothing/clothing.dm

```diff
@@ -1054,26 +1054,17 @@
 	armor = null
 	w_class = ITEMSIZE_NORMAL
 	equip_sound = 'sound/items/equip/jumpsuit.ogg'
-	var/has_sensor = 1 //For the crew computer 2 = unable to change mode
-	var/sensor_mode = 0
-		/*
-		1 = Report living/dead
-		2 = Report detailed damages
-		3 = Report location
-		*/
+	var/has_sensor = SUIT_NO_SENSORS ///0 = No sensors, 1 = Sensors, 2 = Locked sensors
```

Comment:
The DMdoc should not be on the side of the var, but above it, also it uses numerals when there's defines for what this field takes, it should reference defines instead